rusk-wallet/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `data_dir` can be properly overriden [#656]
 
 ## Added
+- Cache notes [#650]
```

Comment:
Should read "Notes cache", that's what we're adding after all: a cache for notes